lib/medtronic/medtronicSimulator.js

```diff
@@ -413,23 +440,29 @@ exports.make = function(config){
       // hasn't already been triggered
       if(suspendingEvent === null) {
         setSuspendingEvent(event);
+      } else {
+        simpleSimulate(event);
       }
-      simpleSimulate(event);
     },
     changeDeviceTime: function(event) {
       simpleSimulate(event);
     },
     finalBasal: function(finalRecordTime) {
```

Comment:
I just haven't got around to it yet - it's on the todo list for remaining work: https://trello.com/c/qkp0Mfj6